composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "datafeedwatch/dfwconnector-magento2",
     "description": "DataFeedWatch Connector for Magento 2",
     "require": {
-        "php": "7.*||~8.1.0||~8.2.0||~8.3.0"
+        "php": "7.*||~8.1.0||~8.2.0||8.3.*"
```

Comment:
@jamal-rivers could you please explain a little more what is the specific case you want to cover with your change?